src/components/SingleArtComments.jsx

```diff
@@ -10,6 +12,7 @@ const SingleArtComments = ({comments}) => {
                 })
             }
         </ul>
+        <CreateComment article_id={article_id} setComments={setComments}/> 
```

Comment:
As a user I should not have to scroll to the very bottom of the comments list to be able to post my own comment.